python/private/py_wheel.bzl

```diff
@@ -40,6 +40,12 @@ Name of the distribution.
 
 This should match the project name onm PyPI. It's also the name that is used to
 refer to the package in other packages' dependencies.
+
+Note that this attribute supports 'make variables'.
```

Comment:
Please update this doc to reflect the current state:
* Makevars aren't expanded
* Workspace status keys (the `{foo}` format, which come from https://bazel.build/docs/user-manual#workspace-status) are expanded

Reply:
I updated the doc to reflect the current state, updated the PR description and the issue details: https://github.com/bazelbuild/rules_python/issues/1250
Please review. Thanks.